ddocs/erlang_filters/packages/traditional-couchapp/build/load.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs')
```

Comment:
Good idea. I was hoping I'd be able to simply release a version with a fix that doesn't look like it's going to happen any time soon.